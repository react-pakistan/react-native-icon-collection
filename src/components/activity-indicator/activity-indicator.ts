import styled from 'styled-components/native';

export const ActivityIndicator = styled.ActivityIndicator``;

export enum ActivityIndicatorSize {
  large = 'large',
  small = 'small',
}
