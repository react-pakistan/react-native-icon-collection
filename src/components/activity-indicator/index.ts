export {
  ActivityIndicator,
  ActivityIndicatorSize,
} from './activity-indicator';
