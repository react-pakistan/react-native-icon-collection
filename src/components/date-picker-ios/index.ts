export { DatePickerIOS } from './date-picker-ios';
