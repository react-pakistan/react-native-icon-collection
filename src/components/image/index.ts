export {
  IImageProps,
  Image,
  ResizeMethod,
  ResizeMode,
} from './image';
