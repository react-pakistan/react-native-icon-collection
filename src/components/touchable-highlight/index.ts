export { TouchableHighlight } from './touchable-highlight';
