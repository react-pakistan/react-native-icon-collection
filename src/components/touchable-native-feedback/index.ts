export { TouchableNativeFeedback } from './touchable-native-feedback';
