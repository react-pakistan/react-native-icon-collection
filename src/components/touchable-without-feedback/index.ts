export { TouchableWithoutFeedback } from './touchable-without-feedback';
