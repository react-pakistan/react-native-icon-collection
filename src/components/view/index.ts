export { View, IView } from './view';
