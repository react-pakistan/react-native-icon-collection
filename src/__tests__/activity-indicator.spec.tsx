import React from 'react';
import renderer from 'react-test-renderer';
import { ActivityIndicator } from '../components/activity-indicator';
import {
  activityIndicatorLargeProps,
  activityIndicatorSmallProps,
} from '../../__stories__/story/activity-indicator.story';

describe('Testing ActivityIndicator component', () => {
  it('matches ActivityIndicator snapshot, small version', () => {
    const wrapper = renderer.create(
      <ActivityIndicator {...activityIndicatorSmallProps} />,
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
  it('matches ActivityIndicator snapshot, large version', () => {
    const wrapper = renderer.create(
      <ActivityIndicator {...activityIndicatorLargeProps} />,
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
