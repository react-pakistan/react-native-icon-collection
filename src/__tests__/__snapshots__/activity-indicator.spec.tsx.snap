// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing ActivityIndicator component matches ActivityIndicator snapshot, large version 1`] = `
<ActivityIndicator
  animating={true}
  color="#00D3CA"
  hidesWhenStopped={false}
  size="large"
  style={
    Array [
      Object {},
    ]
  }
/>
`;

exports[`Testing ActivityIndicator component matches ActivityIndicator snapshot, small version 1`] = `
<ActivityIndicator
  animating={true}
  color="#00D3CA"
  hidesWhenStopped={false}
  size="small"
  style={
    Array [
      Object {},
    ]
  }
/>
`;
